packages/flutter/test/gestures/gesture_binding_resample_event_on_widget_test.dart

```diff
@@ -50,7 +50,7 @@ void main() {
         ui.PointerData(
             change: ui.PointerChange.down,
             physicalX: 0.0,
-            timeStamp: epoch + const Duration(milliseconds: 0),
+            timeStamp: epoch + Duration.zero,
```

Comment:
Here and at L48 I would remove `+ Duration.zero`

Reply:
Great thanks @a14n